api/resolvers/user.js

```diff
@@ -922,11 +922,12 @@ export default {
           createdAt: {
             gte,
             lte
-          }
+          },
+          ...(me?.id === user.id ? {} : { OR: [{ invoiceActionState: 'PAID' }, { invoiceActionState: null }] })
```

Comment:
This can all be an `OR` I think:

e.g.

```
OR: [{userId: me.id}, { invoiceActionState: 'PAID' }, { invoiceActionState: null }]
```

Alternatively, we don't need to count unpaid items at all and it can just be:

```
OR: [{ invoiceActionState: 'PAID' }, { invoiceActionState: null }]
```

Reply:
Decided to never count unpaid items